src/components/SignUp/Account.vue

```diff
@@ -3,45 +3,57 @@
       <div class="account-field">
         <v-text-field
           class="account-inputs"
+          :rules="firstNameRules"
           placeholder="First Name"
           hint="ex: John"
           persistent-hint
+          required
           outlined
         /><span class="asterisk">*</span>
       </div>
       <div class="account-field">
         <v-text-field
           class="account-inputs"
+          :rules="lastNameRules"
           placeholder="Last Name"
           hint="ex: Smith"
           persistent-hint
+          required
           outlined
         /><span class="asterisk">*</span>
       </div>
       <div class="account-field">
         <v-text-field
           class="account-inputs"
+          :rules="emailRules"
           placeholder="Email Address"
           hint="ex: JohnSmith@gmail.com"
           persistent-hint
+          required
           outlined
         /><span class="asterisk">*</span>
       </div>
       <div class="account-field">
         <v-text-field
           class="account-inputs"
+          v-model="userPassword"
+          :rules="passwordRules"
```

Comment:
there should also be a type="password" prop passed into it, and/or eye icon to toggle the mask